mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/WorkerExecutionOperationsNetworkStage.java

```diff
@@ -478,6 +478,12 @@ private void setupSubscriptionStateHandler(ExecuteStageRequest executeStageReque
         };
 
         this.subscriptionStateHandler = subscriptionStateHandler;
+        try {
+            this.subscriptionStateHandler.startAsync().awaitRunning(Duration.of(5, ChronoUnit.SECONDS));
```

Comment:
This should be TimeUnit no?

Reply:
TimeUnit is in java.util.concurrent and this Duration.of requires the temporal unit interface which is from java.time.